document/introduction/index.rst

```diff
@@ -1,6 +1,54 @@
 Introduction
 ============
 
-.. todo::
+WebAssembly (abbreviated Wasm [#wasm]_) is a *safe, portable, low-level code format*
+designed for efficient execution and compact representation.
+Its main goal is to enable high performance applications on the Web, but it does not bake in any Web-specific assumptions or features, so can be employed in other environments as well.
```

Comment:
bake in is a bit informal, so maybe just "make Web-specific assumptions or require Web-specific features".

I won't bikeshed (teehee) the actual text here, but the points lgtm